### ISSUE TITLE
"Unhide" or "Show hidden" button

### ISSUE THREAD

**Issue by repaxan** [dailyscape/dailyscape.github.io#29]
If I accidentally hide something, the way to get it back is by clicking the Reset button, which also resets the order. It would be nice to have something that only resets the hidden items, or shows them as crossed-out or greyed-out rows.

**Comment by 607ch00** [comment]
Yes this is true.. I initially had 2 separate buttons but it felt like the buttons on the top were starting to get crowded a bit and for something that _ideally_ doesn't happen often, I combined it all into one.

I'm not the best at UX stuff, would welcome graphics to show what would be ideal for me to implement without making things too busy and discuss more.

**Comment by irfinnew** [comment]
I also managed to accidentally hide an item that I simply wanted to check as completed.

Perhaps the "hide" action could be moved to the right of the row, so it's not right next to the checkbox? That would make it harder to click accidentally.
